awx/ui_next/src/screens/Project/ProjectList/ProjectListItem.jsx

```diff
@@ -115,34 +121,41 @@ class ProjectListItem extends React.Component {
                   />
                 ) : null}
               </DataListCell>,
-              <DataListCell alignRight isFilled={false} key="sync">
-                {project.summary_fields.user_capabilities.start && (
-                  <Tooltip content={i18n._(t`Sync Project`)} position="top">
-                    <ProjectSyncButton projectId={project.id}>
-                      {handleSync => (
-                        <Button variant="plain" onClick={handleSync}>
-                          <SyncIcon />
-                        </Button>
-                      )}
-                    </ProjectSyncButton>
-                  </Tooltip>
-                )}
-              </DataListCell>,
-              <DataListCell key="edit" alignRight isFilled={false}>
-                {project.summary_fields.user_capabilities.edit && (
-                  <Tooltip content={i18n._(t`Edit Project`)} position="top">
+            ]}
+          />
+          <DataListAction
+            aria-label="actions"
+            aria-labelledby={labelId}
+            id={labelId}
+          >
+            {project.summary_fields.user_capabilities.start && (
+              <Tooltip content={i18n._(t`Sync Project`)} position="top">
+                <ProjectSyncButton projectId={project.id}>
+                  {handleSync => (
                     <Button
+                      css="grid-column: 1"
```

Comment:
I don't really understand specifically setting grid column for these buttons.  shouldn't they just be one after another?

Reply:
I had to add assign each button to a `grid-column` to prevent the action buttons from shifting between columns whenever buttons are hidden due to permissions.

Example: 
<img width="798" alt="Screen Shot 2020-02-12 at 6 37 39 PM" src="https://user-images.githubusercontent.com/15881645/74884349-f617f900-5340-11ea-848a-699812045fa8.png">